settings/src/components/account-status.js

```diff
@@ -79,27 +86,31 @@ export default function AccountStatus() {
  * @param props.status
  * @param props.headerText
  * @param props.bodyText
+ * @param props.disabled
  */
-function SettingStatusCard( { screen, status, headerText, bodyText } ) {
+function SettingStatusCard( { screen, status, headerText, bodyText, disabled = false } ) {
+	const cardContent = (
+		<CardBody>
+			<StatusIcon status={ status } />
+			<h3 aria-label={ 'Click to enter the ' + headerText + ' setting page.' }>
+				{ headerText }
+			</h3>
+			<p>{ bodyText }</p>
+			<Icon icon={ chevronRight } size={ 26 } className="wporg-2fa__status-card-open" />
+		</CardBody>
+	);
+
+	let classes = 'wporg-2fa__status-card wporg-2fa__status-card-' + screen;
+
+	if ( disabled ) {
+		classes += ' is-disabled';
+	}
+
 	return (
-		<Card className={ 'wporg-2fa__status-card wporg-2fa__status-card-' + screen }>
-			<ScreenLink
-				screen={ screen }
-				anchorText={
-					<CardBody>
-						<StatusIcon status={ status } />
-						<h3 aria-label={ 'Click to enter the ' + headerText + ' setting page.' }>
-							{ headerText }
-						</h3>
-						<p>{ bodyText }</p>
-						<Icon
-							icon={ chevronRight }
-							size={ 26 }
-							className="wporg-2fa__status-card-open"
-						/>
-					</CardBody>
-				}
-			/>
+		<Card className={ classes }>
+			{ disabled && cardContent }
```

Comment:
I'm pretty sure you could do this with a ternary:
```
{ disabled ? cardContent : <ScreenLink /> }
```

Reply:
That does work, but IMO the readability is lower:

`{ disabled ? cardContent : <ScreenLink screen={ screen } anchorText={ cardContent } /> }`

I feel like there's too much going on in that line to skim it quickly, I have to slow down to pull the different sections apart. I think the form in [9973b5f](https://github.com/WordPress/wporg-two-factor/pull/157/commits/9973b5f7f07bd6e3e7e1916db0c67a32de28cf37) has that problem to a lesser extent, but `prettier` wanted it this way instead of allowing the attributes to have some breathe room on their own line.

```js
<Card className={ classes }>
	{ disabled && cardContent }

	{ ! disabled &&
		 <ScreenLink 
		 	screen={ screen }
			anchorText={ cardContent }
		/>
	}
</Card>
```

Reply:
It's pretty common to use a ternary for these states in react projects. You'll find it used often in Gutenberg and Gutenberg documentation ([example](https://github.com/WordPress/gutenberg/blob/trunk/docs/getting-started/create-block/author-experience.md)).

Reply:
Yeah I agree this is a very common React pattern

Reply:
Yeah, I don't doubt it's common in general; this specific case just feels harder to read to me when doing it that way, and there isn't anything in [the Gutenberg guidelines](https://github.com/WordPress/gutenberg/blob/f8d74a6230b81a7980f492ba5c1a38cc23f28315/docs/contributors/code/coding-guidelines.md) or `prettier` about it. I went ahead and did it in becdb2c so we can get this out the door though.

---- 

More broadly, I feel like these types of discussions add a disproportionate amount of delay and debate to PRs, compared to the potential benefit of a minor style tweak. What do y'all think about an approach where we [agree on standards as a team once, and then let the tooling do the work of ensuring conformance](https://www.mattlayman.com/blog/2017/no-nitpicking-code-reviews/)? We could add a custom rule to the Gutenberg standards in a different PR if the team wants to.